### ISSUE TITLE
A11y_AzureDataStudio(Schema,Dacpac,Backup)_Schema Compare Options_SchemaCompare_Keyboard: Tab items are not accessible using keyboard.

### ISSUE THREAD

**Issue by ShwetaBaglane** [microsoft/azuredatastudio#6946]
[“Check out Accessibility Insights! !](https://accessibilityinsights.io/) - Identify accessibility bugs before check-in and make bug fixing faster and easier.”

**Environment Details:**
Application Name: Azure Data Studio
Application Version: 1.10.0-insider (user setup)
Commit: 1ff9a2e
Date: 2019-08-09T07:49:00.254Z
VS Code 1.37.0
Electron: 4.2.7
Chrome: 69.0.3497.128
Node.js: 10.11.0
V8: 6.9.427.31-electron.0
OS: Windows_NT x64 10.0.18362

**Additional Details:**
MAS References: MAS2.1.1

**Repro Steps:**
1. Launch Azure Data Studio application.
2. Connect to server with windows credentials.
3. Navigate to Extensions from left menu.
4. Search & install Schema Compare extension.
5. Navigate to Connections from left menu.
6. Expand the server connected in step 2.
7. Right click an on prem database and select Schema compare from Context menu.
8. Navigate till Select Source or Select Target & hit enter.
9. Provide all the details for Source & Target.
10. Navigate till Ok button & hit enter.
11. Navigate till Options menu from Schema compare Menu bar & hit enter.
12. Navigate till tab items using tab key and press enter/ space key.

**Actual:**
Tab items are not accessible using Keyboard. When pressed space key action performed and when pressed enter key, dialog gets closed.

**Expected:**
Tab items should be accessible using keyboard.

**Recommendations:**
Refer below link which is repository of bug fixes code snippets: (https://microsoft.sharepoint.com/teams/msenable/mas/pages/browse-fixes.aspx)

**User Impact:**
Keyboard and screen reader only users will not be able to access tab items as it is not accessible using keyboard.

**MAS Reference:**
MAS 2.1.1
https://microsoft.sharepoint.com/:w:/r/teams/msenable/_layouts/15/WopiFrame.aspx?sourcedoc={8492c4eb-c179-40ae-8777-cd044ed725a2}

**Attachment for Reference:**
[A11y_AzureDataStudio(Schema,Dacpac,Backup)_Schema Compare Options_SchemaCompare_Keyboard.pptx](https://github.com/microsoft/azuredatastudio/files/3539833/A11y_AzureDataStudio.Schema.Dacpac.Backup._Schema.Compare.Options_SchemaCompare_Keyboard.pptx)

**Comment by fsteffi** [comment]
Checked and verified in below version the bug is no longer reproduced and hence closing the bug.

Version Details:Version: 1.12.0-insider
Commit: 8e1a2248e47925eb825b0ce037b96b6c87798a8a
Date: 2019-10-04T07:52:03.973Z
VS Code: 1.38.0
Electron: 4.2.10
Chrome: 69.0.3497.128
Node.js: 10.11.0
V8: 6.9.427.31-electron.0
OS: Windows_NT x64 10.0.18362
[#6946A11y_AzureDataStudio(Schema,Dacpac,Backup)_Schema Compare Options_SchemaCompare_Keyboard.pptx](https://github.com/microsoft/azuredatastudio/files/3712056/6946A11y_AzureDataStudio.Schema.Dacpac.Backup._Schema.Compare.Options_SchemaCompare_Keyboard.pptx)
